index.html

```diff
@@ -4683,82 +4683,134 @@ <h2>Internationalization Considerations</h2>
     </p>
 
     <section class="informative">
-      <h3>Language Declarations</h3>
+      <h3>Language and Direction</h3>
 
       <p>
-When expressing human-readable text content in the data model, it is important
-for presentation, accessibility, and further processing to specify the language
-of the text, as follows:
+Data publishers are strongly encouraged to use the design pattern described
+in this section when expressing simple internationalized strings in
+JSON, JSON-LD, or similar data formats to ensure the broadest interoperability
+across syntaxes. The design pattern is a result of extensive collaboration
+between the W3C Internationalization Activity, the Linked Data Community, and
+developers that primarily utilize JSON and CBOR to publish information.
       </p>
 
-      <ul>
-        <li>
-[[!JSON-LD]] supports the expression of language information using the
-<code>@language</code> feature, as described in [[!JSON-LD]] 1.1,
-<a href="https://www.w3.org/TR/json-ld11/#string-internationalization">Section 4.2.3: String Internationalization</a>.
-        </li>
-        <li>
-[[!JSON-LD]] also supports the expression of language information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
+      <p>
+The general design pattern is to use the following markup template when
+expressing a text string that is tagged with a language and, optionally, a
+specific text direction:
+      </p>
+
+      <pre class="example nohighlight" title="Design pattern for internationalized text strings">
+"<a>property</a>": {
+  "value": "<span class="highlight">The string value</span>",
+  "language: "<code>LANGUAGE</code>-x-d-<code>DIRECTION</code>"
```

Comment:
Does this section also need to show and address the [language map](https://www.w3.org/TR/json-ld/#dfn-language-map) feature of JSON-LD?

Reply:
We don't have enough field experience with language maps being used for Verifiable Credentials and thus I feel that it's a bit premature to provide any guidance there. We are doing some work with governments and Verifiable Credentials that cross international boundaries so will be experimenting with this over the next couple of years to see what the best mode of operation is (an array of langtag'ed content, or a language map).

One potential downside here is that the `-x-dir-` mechanism might throw a wrench into using language maps... because now you have to look for `obj.title["ar-x-dir-rtl"]` as well as `obj.title["ar-x-dir-rtl"]`. In any case, don't think we'll solve that one any time soon...

Reply:
I don't understand the "potential downside"?

The `-x-dir` mechanism comes at the end of the language tag, so BCP47 Lookup matching (or other matching such as ES-402 best fit) can still easily be applied. In most cases, we want any extensions to be ignored.

Reply:
Can discuss more on Wed call. The downside here is that plain 'ol JSON developers will have a larger number of variations to consider if the `-x-dir-` or `-d-` mechanism is used.

Reply:
Based on the calls w/ the i18n, JSON-LD, and VC communities, it looks like we're dropping x-dir- and -d- in favor of doing the right long term fix.